Clockwise/ConfigurationExtensions.cs

```diff
@@ -93,7 +105,8 @@ IDisposable Subscribe(HandleCommand<TCommand> handle, Func<HandleCommand<TComman
                     return subscribe(async delivery =>
                     {
                         {
-                            var stateDescriptor = await cb.GetLastStateAsync();
+                            var cb = await circuitBreaker.Value;
```

Comment:
This side effect expectation on a closure from `AfterCreating` looks fragile. Let's work on this one a bit.

Reply:
feel free to sketch something better, not sure if I can get to this sooner than Tuesday